ansible/ansible.cfg

```diff
@@ -3,7 +3,7 @@ inventory = hosts.yaml
 error_on_undefined_vars = false
 timeout = 30
 forks = 10
-connection_plugins = plugins/connection/
+connection_plugins = community.general.incus
```

Comment:
I think we can just push a commit to clear the plugins stuff as the Incus connection plugin is widely available nowadays.

Reply:
I see you actually did delete the whole folder, so I don't think the line in the .cfg is needed anymore.